client/src/utils/index.js

```diff
@@ -1,4 +1,4 @@
 /* eslint-disable import/prefer-default-export */
 export function sanitize(str) {
-  return str.replace(/[^A-Za-z0-9]/g, '-').replace(/[<>]/ig, '')
+  return str.replace(/[^A-Za-z0-9._]/g, '-').replace(/[<>]/ig, '')
```

Comment:
I can't remember the reason for stripping out non-alphanumeric characters, but could you remind me the reason for this change?

Reply:
This download filename (which is sanitized) is used when you click on the link to download the file. If you remove `.` the downloaded file name is like `mypicturejpg` instead of `mypicture.jpg`. 

I think at least `.` should be kept as some system use the extension to identify file. I also add `_` more for practical reason.

I don't  see any security issue but I can miss something. An other option would be to keep an extra field only for original filename ?